src/chainstate/burn/operations/mod.rs

```diff
@@ -308,6 +316,20 @@ pub struct DelegateStxOp {
     pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
 }
 
+#[derive(Debug, PartialEq, Clone, Eq, Serialize, Deserialize)]
+pub struct PegInOp {
+    pub recipient: PrincipalData,
+    pub peg_wallet_address: PoxAddress,
+    pub amount: u64,   // BTC amount to peg in, in satoshis
+    pub memo: Vec<u8>, // extra unused byte
```

Comment:
This field can be more than 1 byte long. Please update the comment.

Reply:
Oops, thanks for spotting. Fixed!